src/client/game.cpp

```diff
@@ -1280,9 +1280,9 @@ bool Game::createSingleplayerServer(const std::string &map_dir,
 	}
 
 	if (bind_addr.isIPv6() && !g_settings->getBool("enable_ipv6")) {
-		*error_message = "Unable to listen on " +
+		*error_message = gettext("Unable to listen on ") +
 				bind_addr.serializeString() +
-				" because IPv6 is disabled";
+				gettext(" because IPv6 is disabled");
```

Comment:
This must be turned into a formatted string, like this:
```cpp
    const char *template = gettext("Unable to listen on %s because IPv6 is disabled");
    char buf[128];
    porting::mt_snprintf(buf, sizeof(buf), template, bind_addr.serializeString().c_str());
    *error_message = buf;
```